plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveBucketHandle.java

```diff
@@ -83,7 +93,7 @@ public HiveBucketProperty toTableBucketProperty()
                         .collect(toList()),
                 bucketingVersion,
                 tableBucketCount,
-                ImmutableList.of());
```

Comment:
why it was empty before? Was this a bug?

Reply:
might be because it was not needed until now, keeping it empty doesn't seem to cause any problems in existing code

Reply:
I think it's used in read path only